examples/fastai/train.py

```diff
@@ -0,0 +1,42 @@
+#
+# This short example is based on the fastai GitHub Repository of vision examples
+# https://github.com/fastai/fastai/blob/master/examples/vision.ipynb
+# Modified here to show mlflow.fastai.autolog() capabilities
+#
+import argparse
+from fastai.vision import *
```

Comment:
I think we should avoid using `import *`. Can we just import `untar_data`, `ImageDataBunch`, and `cnn_learner`?

Reply:
You need to import all these: `untar_data, ImageDataBunch, models, URLs, imagenet_stats, rand_pad, cnn_learner, accuracy`

Is that better than `import *`?

Reply:
Or you could do something like:

```python
import fastai.vision as vis

vis.untar_data(...)
vis.ImageDataBunch(...)
```

Reply:
Indeed, much cleaner and pythonic.